src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family in case 'Roboto' fails to load for some reason. For example, 'sans-serif' can be a good fallback.